### ISSUE TITLE
asset download should give !=0 when no assets found/other error happens

### ISSUE THREAD

**Issue by jupe** [j0057/github-release#60]
Tool should return non success return code when asset download fails or no assets found with used PATTERN. This would simplify/help scripting..

**Comment by lyoshenka** [comment]
Agreed. I use githubrelease as part of CI and if it doesn't return an error code when the asset is not found, the CI system won't know that something went wrong.
